### ISSUE TITLE
Managed server failed to start due to boot identity problem

### ISSUE THREAD

**Issue by jhzhu89** [oracle/weblogic-kubernetes-operator#326]
Hi,

I created a domain, but  all managed server always failed to start due to below error:

```
1. weblogic.security.SecurityInitializationException: Authentication denied: Boot identity not valid. The user name or password or both from the boot identity file (boot.properties) is not valid. The boot identity may have been changed since the boot identity file was created. Please edit and update the boot identity file with the proper values of username and password. The first time the updated boot identity file is used to start the server, these new values are encrypted.
2. java.lang.IllegalStateException: Unable to perform operation: post construct on weblogic.security.SecurityService
3. java.lang.IllegalArgumentException: While attempting to resolve the dependencies of weblogic.jndi.internal.RemoteNamingService errors were found
4. java.lang.IllegalStateException: Unable to perform operation: resolve on weblogic.jndi.internal.RemoteNamingService
5. java.lang.IllegalArgumentException: While attempting to resolve the dependencies of weblogic.deployment.DeploymentRegistrationService errors were found
6. java.lang.IllegalStateException: Unable to perform operation: resolve on weblogic.deployment.DeploymentRegistrationService 
```

Could you help check what is wrong with the boot identity? Thanks in advance.

Here is all files inside the domain:
[domain1.tar.gz](https://github.com/oracle/weblogic-kubernetes-operator/files/2074714/domain1.tar.gz)

**Comment by rjeberhard** [comment]
@jhzhu89, what can you tell us about how you created this domain home?  The boot.properties files between the admin server, managed server 1, and managed server 3 are all different.  It's likely the boot.properties for managed servers 1 and 2 are wrong.

**Comment by karan-kapoor90** [comment]
I killed the managed server pods and put in the admin username and password into the boot.properties for both the managed servers.  These are the same credentials I gave in the secret for weblogic credentials. 

In my case the operator had hit the max number of tries and had hence given up on creating managed servers, so I restarted an operator pod which immediately triggered the managed server creation. However the new pods also has the same issue. I could be wrong but assuming that the creds should look the same after AES encryption, I cross references the boot.properties from 2 managed servers and the admin server, but all 3 were different.

At this point, problem still stands unresolved.
